client/src/components/Buttons.js

```diff
@@ -8,27 +8,35 @@ const Buttons = () => {
     setTotalVotes(totalVotes + 1);
   };
   const decreaseVote = () => {
-    if (totalVotes >= 0) {
+    if (totalVotes > 0) {
       setTotalVotes(totalVotes - 1);
     }
   };
 
   return (
     <div>
       <div className="btn-container">
-        <FontAwesomeIcon
-          icon={faThumbsUp}
-          size="3x"
-          onClick={increaseVote}
-          style={{ color: "green" }}
-        />
+        <button>
+          <FontAwesomeIcon
+            aria-label="button"
+            role="button"
+            icon={faThumbsUp}
+            size="3x"
+            onClick={increaseVote}
+            style={{ color: "green" }}
+          />
+        </button>
```

Comment:
this is good, much better as a button! With your aria-label here if you instead add it to the button element and give it a name which conveys meaning to screen readers (as there is only an icon and no descriptive text), for example here `aria-label="upvote"` would work